club-website/src/components/header.tsx

```diff
@@ -0,0 +1,62 @@
+"use client"
+import * as React from "react"
+import Link from "next/link"
+import Image from "next/image";
+import {
+    NavigationMenu,
+    NavigationMenuContent,
+    NavigationMenuIndicator,
+    NavigationMenuItem,
+    navigationMenuTriggerStyle,
+    NavigationMenuLink,
+    NavigationMenuList,
+    NavigationMenuTrigger,
+    NavigationMenuViewport,
+  } from "@/components/ui/navigation-menu"
+
+export default function Header() {
+
+ 
+    return (
+        
+        <div className="flex justify-between p-6" id="logo_address">
+            <Link href="/about" legacyBehavior passHref>
+                <Image id="logo" src="/favicon180.png" alt="" width={50} height={50}></Image> 
```

Comment:
not sure if i'm correct but should the logo on the header match the logo in the figma?

Reply:
done!